### PR TITLE
Footer-Header updated

### PR DESCRIPTION
**Pull request by BilalAli-01**
added footer to all pages
removed underline when hovering
changed colour when hovered
changed colour when active
animation on logo when hovered

### CODE REVIEW

**Comment by vongb** [comment]
Nice logo hover animation!

File: frontend/src/components/SiteHeader.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Container, Button } from "react-bootstrap";
```
Comment (Aaron-K-T-Berry) [inline frontend/src/components/SiteHeader.tsx]: Remove unused import `Button`

File: frontend/src/components/SiteHeader.tsx [hunk 2]
```diff
@@ -18,22 +18,22 @@ export default class SiteHeader extends React.Component<
         <Navbar>
           <Container>
             <Row>
-              <Link to="/">
-                <h1><img src="./placeholder_assets/logo.jpg" width="50" height="50" ></img>Buy</h1>
+              <Link style={{ textDecoration: 'none' }} to="/">
+                <h1 className="logo-img"><img src="./placeholder_assets/logo.jpg" width="50" height="50" ></img>Buy</h1>
```
Comment (Aaron-K-T-Berry) [inline frontend/src/components/SiteHeader.tsx]: images should have `alt` text as per A11Y standards https://a11yproject.com/posts/alt-text/

File: frontend/src/components/SiteFooter.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import React from "react";
 import "../styles/App.css";
+import { Link } from "react-router-dom";
```
Comment (Aaron-K-T-Berry) [inline frontend/src/components/SiteFooter.tsx]: Remove unused import `Link`

File: frontend/src/components/SiteFooter.tsx [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
+import { Button } from "react-bootstrap";
```
Comment (Aaron-K-T-Berry) [inline frontend/src/components/SiteFooter.tsx]: Remove unused import `Button`

**Review by Aaron-K-T-Berry — DISMISSED** [review]

**Comment by Aaron-K-T-Berry** [comment]
@BilalAli-01 CSS changes are looking great 👍

**Review by BilalAli-01 — COMMENTED** [review]
thanks

**Review by Aaron-K-T-Berry — APPROVED** [review]
LGTM

**Pull request merged**